source/lac/petsc_matrix_base.cc

```diff
@@ -307,9 +307,9 @@ namespace PETScWrappers
   MatrixBase::size_type
   MatrixBase::local_size() const
   {
-    PetscInt n_rows, n_cols;
+    PetscInt n_rows;
 
-    const PetscErrorCode ierr = MatGetLocalSize(matrix, &n_rows, &n_cols);
+    const PetscErrorCode ierr = MatGetLocalSize(matrix, &n_rows, nullptr);
```

Comment:
It's not wrong, I guess, but why do this change?

Reply:
because `n_cols` is unused by this method.

Reply:
Oh, I must have read this change in the wrong direction :-( Thanks for pointing out the obvious!